paint/PAINT_Tool.h

```diff
@@ -2,13 +2,14 @@
 #include <iostream>
 #include <vector>
 
-struct SDL_Renderer;
-struct SDL_Texture;
+namespace win
+{
+	struct Coords;
+}
 
 namespace paint
 {
 	class Brush;
-	struct Coords;
 	struct Line
```

Comment:
Line should probably be moved to gfx. Particularly if you implement clipping, because then you'll need to write some less than trivial code to compute intersections.
**SHOULD**

Reply:
done